io-engine/src/bdev/nvmx/uri.rs

```diff
@@ -227,6 +227,12 @@ impl<'probe> NvmeControllerContext<'probe> {
             )
             .with_transport_retry_count(
                 Config::get().nvme_bdev_opts.transport_retry_count as u8,
+            )
+            .with_fabrics_connect_timeout_us(
+                crate::subsys::config::opts::try_from_env(
+                    "NVMF_FABRICS_CONNECT_TIMEOUT",
+                    3_000_000,
```

Comment:
Probably now we can leave this as 500ms by default since we uncovered the real culprit?

```suggestion
                    500_000,
```

Reply:
I've left it as 1s for now, just to give some more slack just in case.